contracts/nois-proxy/src/contract.rs

```diff
@@ -262,7 +262,7 @@ fn execute_withdraw(
 pub fn sudo(deps: DepsMut, env: Env, msg: SudoMsg) -> Result<Response, ContractError> {
     match msg {
         #[cfg(feature = "governance_owned")]
-        SudoMsg::Withdaw {
+        SudoMsg::withdraw {
```

Comment:
Oh nice, you added a bug here (lower case) which my following PR found. This is why main is currently red. Will fix.